codesign/bin/codesign.dart

```diff
@@ -6,35 +6,41 @@ import 'dart:io';
 
 import 'package:args/args.dart';
 import 'package:codesign/codesign.dart';
+import 'package:file/file.dart';
+import 'package:file/local.dart';
+import 'package:logging/logging.dart';
 import 'package:platform/platform.dart';
+import 'package:process/process.dart';
 
 /// Definitions of variables are included in help texts below.
 const String kHelpFlag = 'help';
-const String kCommit = 'commit';
-const String kProduction = 'production';
-const String kCodesignCertName = 'codesign-cert-name';
-const String kCodesignUserName = 'codesign-username';
-const String kAppSpecificPassword = 'app-specific-password';
-const String kCodesignAppStoreId = 'codesign-appstore-id';
-const String kCodesignTeamId = 'codesign-team-id';
-const String kCodesignFilepath = 'filepath';
+const String kCommitOption = 'commit';
+const String kProductionFlag = 'production';
+const String kCodesignCertNameOption = 'codesign-cert-name';
+const String kCodesignUserNameOption = 'codesign-username';
+const String kAppSpecificPasswordOption = 'app-specific-password';
+const String kCodesignAppStoreIdOption = 'codesign-appstore-id';
+const String kCodesignTeamIdOption = 'codesign-team-id';
+const String kCodesignFilepathOption = 'filepath';
 
 /// Perform Mac code signing based on file paths.
 ///
-/// If `--production` is set to true, code signed artifacts will be uploaded
+/// If ```--production``` is set to true, code signed artifacts will be uploaded
 /// back to google cloud storage.
 /// Otherwise, nothing will be uploaded back for production. default value is
 /// false.
 ///
-/// For `--commit`, asks for the engine commit to be code signed.
+/// For ```--commit```, asks for the engine commit to be code signed.
 ///
-/// For `--filepath`, provides the artifacts zip paths to be code signed.
+/// For ```--filepath```, provides the artifacts zip paths to be code signed.
 ///
 /// Usage:
+/// ```
```

Comment:
nit: ```shell

This'll indicate it's not meant as a code sample

Reply:
Thanks for explaining! Yeah I misunderstood the feedback from https://github.com/flutter/cocoon/pull/1972#discussion_r920133129 and changed the wrong sticks.